ARGUS/ARGUS_Transforms.py

```diff
@@ -95,13 +95,7 @@ def randomize(self, data: NdarrayOrTensor) -> None:
                     slice_roi[self.axis] = self._roi_center_slice
                     is_labeled = (data[tuple(slice_roi)]!=0).any()
                     if is_labeled:
-                        slice_roi[self.axis] -= self.boundary
-                        is_labeled = (data[tuple(slice_roi)]!=0).any()
-                        if is_labeled:
-                            slice_roi[self.axis] += self.num_slices-1
-                            is_labeled = (data[tuple(slice_roi)]!=0).any()
-                            if is_labeled:
-                                break
+                        break

```

Comment:
LGTM! I was thinking only a single slice was being checked. Thanks for making this change. It is an important one.